source/Engine/Media/Decoders/AudioDecoder.cpp

```diff
@@ -18,10 +18,11 @@ struct AudioPacket {
 // Lifecycle functions
 AudioDecoder::AudioDecoder(MediaSource* src, int stream_index) {
 	int ret;
-	if (stream_index < 0) {
-		Log::Print(Log::LOG_ERROR, "stream_index < 0");
-		return;
-	}
+	ERROR_RET_VOID(stream_index >= 0, "stream_index < 0");

```

Comment:
```suggestion
	VERB_ERROR_RET_VOID(stream_index >= 0, "stream_index < 0");
```
Doesn't really seem like this can happen, looking at `source/Engine/Media/MediaPlayer.cpp`.